language/move-lang/src/typing/core.rs

```diff
@@ -288,6 +288,16 @@ impl Context {
         }
     }
 
+    fn is_a_friend_of(&self, m: &ModuleIdent) -> bool {
```

Comment:
nit, I'd rename this to `current_module_is_a_friend_of` The context isn't a friend of anyone :P